src/wasmtime.rs

```diff
@@ -59,26 +63,36 @@ including calling the start function if one is present. Additional functions
 given with --invoke are then called.
 
 Usage:
-    wasmtime [-odg] <file>...
-    wasmtime [-odg] <file>... --invoke=<fn>
+    wasmtime [-odg] [--preload=<wasm>...] [--env=<env>...] [--dir=<dir>...] [--mapdir=<mapping>...] <file> [<arg>...]
+    wasmtime [-odg] [--preload=<wasm>...] [--env=<env>...] [--dir=<dir>...] [--mapdir=<mapping>...] --invoke=<fn> <file> [<arg>...]
     wasmtime --help | --version
 
 Options:
     --invoke=<fn>       name of function to run
     -o, --optimize      runs optimization passes on the translated functions
     -g                  generate debug information
     -d, --debug         enable debug output on stderr/stdout
+    --preload=<wasm>    load an additional wasm module before loading the main module
```

Comment:
Maybe allow multiple modules to be preloaded?

Reply:
You can use --preload= multiple times.

Reply:
Didn't know that.